src/os/bluestore/BlueStore.cc

```diff
@@ -5651,6 +5653,29 @@ ObjectMap::ObjectMapIterator BlueStore::get_omap_iterator(
 
 // -----------------
 // write helpers
+void BlueStore::_save_min_min_alloc_size(uint64_t new_val)
+{
+  assert(new_val > 0);
+  if (min_min_alloc_size > 0 &&
+      new_val >= min_min_alloc_size) {
```

Comment:
if new_val > min_min_alloc_size we want to warn with something like

"min_min_alloc_size $minmin < min_alloc_size $min, may impact performance"